src/Figlet/Figlet.php

```diff
@@ -832,6 +832,10 @@ protected function _smushAmount()
             if (empty($leftChar) || $leftChar === ' ') {
                 $amount++;
             } elseif (! empty($rightChar)) {
+                /**
+                 * @psalm-suppress RedundantConditionGivenDocblockType
```

Comment:
We should baseline these, not suppress them

Reply:
Made all the suggested changes and pushed the code. Kindly review. Thank you.